tests/torch/test_extensions_build.py

```diff
@@ -34,9 +35,12 @@ def test_force_cuda_build(tmp_path):
             pytest.skip('There is no CUDA on the machine. The test will be skipped')
 
     torch_build_dir = tmp_path / 'extensions'
-    export_env_variables = "export CUDA_VISIBLE_DEVICES='' export TORCH_EXTENSIONS_DIR={}".format(torch_build_dir)
 
-    python_executable_with_venv = ". {0}/bin/activate && {1} && {0}/bin/python".format(venv_path, export_env_variables)
+    export_env_variables = [
+        "CUDA_VISIBLE_DEVICES=''",
+        "TORCH_EXTENSIONS_DIR={}".format(torch_build_dir)
+    ]
```

Comment:
Now the `tests.shared.command.Command` has an `env` constructor argument that you could pass. It is then passed to `subprocess.Popen` and the env vars are being set automatically in a platform-specific way. I think it is better to use this than to bake env variables into the command line.